Content.Server/GameObjects/EntitySystems/PumpSystem.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Content.Server.GameObjects.Components.Atmos;
+using Robust.Shared.GameObjects.Systems;
+
+namespace Content.Server.GameObjects.EntitySystems
+{
+    public class PumpSystem : EntitySystem
+    {
+        public override void Update(float frameTime)
+        {
+            base.Update(frameTime);
+            foreach (var pump in ComponentManager.EntityQuery<BasePumpComponent>())
+            {
+                pump.Update(frameTime);
+            }
+        }
+    }
+}
```

Comment:
It'd probably be best if all of pipenet updates were on their own update stage in GridAtmosphereComponent, and have them update only the pipenets on the current grid.